charts/timescaledb-single/admin-guide.md

```diff
@@ -126,17 +126,16 @@ my-release-replica   ClusterIP      None            <none>                     5
 
 ## Cleanup
 
-Removing a deployment can be done by deleting a Helm deployment, however, removing the deployment does not remove:
-- the Persistent Volume Claims (pvc) belonging to the cluster
-- or the headless service that is used by Patroni for its configuration
+Removing a deployment can be done by deleting a Helm deployment, however, removing the deployment does not remove
+the Persistent Volume Claims (pvc) belonging to the cluster.
 
 To fully purge a deployment in Kubernetes, you should do the following:
 ```console
 # Delete the Helm deployment
 helm delete my-release
```

Comment:
Maybe helm delete --purge my-release so the name can be reused(https://stackoverflow.com/questions/51594064/what-is-the-purpose-of-helm-delete-purge)

Reply:
Hmm, helm 3 does not have that flag anymore:
```console
$ helm delete --purge
Error: unknown flag: --purge
```

Reply:
Ah okay, should we insist our customers use helm 3?

Reply:
These Charts are public, so many versions of Helm will be used I think.
They do however work with both Helm 2 and Helm 3, I think it best to keep compatibility as long as possible.

Reply:
Nit: I looked into helm 3 and it seems like `helm uninstall` has the functionality we'd like.

Reply:
Nit: should it be `helm uninstall` for v3? https://helm.sh/docs/intro/using_helm/#helm-uninstall-uninstalling-a-release

Reply:
Currently the charts work with both Helm 2 and Helm 3, i'd like it to keep it that way as long as possible. Helm 2 does not understand `uninstall`, whereas `delete` is an alias for `uninstall` in Helm 3.